core/include/tee/tee_fs.h

```diff
@@ -66,18 +67,19 @@ struct tee_fs_dirent {
 	char *d_name;
 };
 
+
 /*
- * tee_fs implemets a POSIX like secure file system
+ * tee_fs implemets a POSIX like secure file system with GP extension
  */
 struct tee_file_operations {
-	int (*open)(const char *file, int flags, ...);
+	int (*open)(TEE_Result *errno, const char *file, int flags, ...);
```

Comment:
Why aren't we returning the TEE_Result as most other functions providing a TEE_Result does?